sdk/ai/azure-ai-generative/azure/ai/generative/evaluate/metrics/_custom_metric.py

```diff
@@ -2,22 +2,16 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 import abc
-import importlib
-import json
-import os
-import urllib.parse
-
-from azure.ai.generative.evaluate.metrics._aggregators import mean
-from azure.ai.generative.evaluate.metrics._parsers import ScoreReasonParser, ScoreParser
 
 
 class Metric(metaclass=abc.ABCMeta):
 
-    def __init__(self, name):
+    def __init__(self, *, name, description, **kwargs):
         self.name = name
+        self.description = description
 
 
-class CodeMetric(Metric):
+class _CodeMetric(Metric):
```

Comment:
_custom_metric.py is already a private namespace. What's the point of prefix an underscore?